modules/mobile/app/controllers/mobile/v0/payment_history_controller.rb

```diff
@@ -5,6 +5,8 @@
 module Mobile
   module V0
     class PaymentHistoryController < ApplicationController
+      before_action { authorize :bgs, :access? }
```

Comment:
Adding this per discussion with @jperk51. The auth services used this authorization and the web controller does too so we figured we just missed the requirement.